Platform/CommonBoardPkg/Tools/GpioDataConvert.py

```diff
@@ -564,10 +564,12 @@ def get_sbl_dws (inp_fmt, cfg_file, parts):
     sbl_dw0 = SBL_DW0 ()
     sbl_dw1 = SBL_DW1 ()
 
-    pad_name = normalize_pad_name (parts[0])
-    pad_name = pad_name[-7:]
-    if not pad_name.startswith('GPP_'):
+    splitdata = parts[0].split('_')
+    pin_number = splitdata[len(splitdata)-1]
+    pin_number = normalize_pad_name (pin_number)
+    if not 'GPP' in parts[0]:

```

Comment:
Can we serach for 'GPP_' and not 'GPP', and also move this check to before Line #567. We dont want to normalize at all if there is no 'GPP_'.

Reply:
Any update here?

Reply:
I will update the patch today